### ISSUE TITLE
Should treat tool specs as case insensitive

### ISSUE THREAD

**Issue by JohnnyMorganz** [LPGhatguy/aftman#41]
Right now, aftman treats e.g. `UpliftGames/wally@0.3.1` and `upliftgames/wally@0.3.1` as separate tools and will install the latter again even if the former is already installed. They should instead be treated as the same tool.

I'm unsure however if there are any negative side effects for it being case insensitive

**Comment by LPGhatguy** [comment]
Yeah, this is definitely something to fix. We can use manually implement `PartialEq` and `Hash` to use lowercase variants of names while preserving the original case.

**Comment by LPGhatguy** [comment]
Aftman is being archived. Please see LPGhatguy/aftman#74 for more information and alternatives!
